File: test/test_plantToCode/test_to_python/test_system/ultimate_example/orc.py
from i_attack import IAttack
from i_walk import IWalk


class Orc(IAttack, IWalk):

	def __init__(self, name, age):
		self.name = name
		self.__age = age

	def attack(self, damage):
		pass

	def __sleep(self, hours):
		pass

	def walk(self):
		pass

File: test/test_plantToCode/test_to_java/test_system/ultimate_example/Orc.java
import java.util.*;

public class Orc implements IAttack, IWalk{

    public String name;
    private int age;
    private Weapon weapon;

    public void attack(int damage) {
        throw new UnsupportedOperationException();
    }

    public void walk() {
        throw new UnsupportedOperationException();
    }

    private void sleep(int hours) {
        throw new UnsupportedOperationException();
    }
}